packages/app/src/views/Home.vue

```diff
@@ -1,27 +1,34 @@
 <template>
   <div class="m-auto">
     <pre>{{ user }}</pre>
+    <p>Suchefeld</p>
+    <Textfield class="mb-4" v-model="text" icon="search" placeholder="Titelsuche"></Textfield>
+    <p>Textfield</p>
+    <Textfield class="mb-4" v-model="text" label="Title" placeholder="Titelsuche"></Textfield>
```

Comment:
```suggestion
    <Textfield class="mb-4" v-model="text" label="Title" placeholder="Titelsuche"/>
```

Reply:
fixed in 43ad87abf73eacdd5789a326654f80382c57979e

Reply:
fixed in 43ad87abf73eacdd5789a326654f80382c57979e